scripts/ci.js

```diff
@@ -129,9 +174,17 @@ const main = () => {
 };
 
 if (require.main === module) {
-  main().catch(err => {
-    console.error(err);
+  (async () => {
+    try {
+      await main();
+    } catch (err) {
+      console.error(err);
+      // eslint-disable-next-line no-process-exit
+      process.exit(1);
+    }
+
+    // CloudFormation waitFor may still be running so force an exit
```

Comment:
Other note -- is there a way we can just cancel all the other promises? I've suggested `Promise.race` which just returns early with first resolved/rejected one.

Last I remember there was no good way to cancel things like would be useful for us, but thought I'd ask!

Reply:
It’s the listeners the underlying AWS SDK have setup I believe that cause this to hang. I didn’t see a way to deregister the listeners from looking through the docs.

Reply:
It’s the listeners the underlying AWS SDK have setup I believe that cause this to hang. I didn’t see a way to deregister the listeners from looking through the docs.

Reply:
Gotcha. Maybe a quick code comment about that too?